### PR TITLE
New Book abstract and Anthology books

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
Content "Adds anthology book type to increase variety"

#### Purpose of change
Increase book variety

#### Describe the solution
Adds Anthology abstract several new anthologies and adds them to itemgroups for spawning.

#### Describe alternatives you've considered
none

### CODE REVIEW

File: data/json/itemgroups/SUS/library.json [hunk 1]
```diff
@@ -204,7 +204,7 @@
       { "item": "novel_thriller", "count": [ 1, 5 ], "prob": 80 },
       { "item": "novel_crime", "count": [ 1, 5 ], "prob": 80 },
       { "item": "novel_horror", "count": [ 1, 5 ], "prob": 80 },
-      { "item": "novel_tragedy", "count": [ 1, 5 ], "prob": 80 }
+      { "item": "novel_tragedy", "count": [ 1, 5 ], "prob": 80 },
```
Comment (LaVeyanFiend) [inline data/json/itemgroups/SUS/library.json:207]: Comma at last entry bad(GitHub isn't letting me do the code suggestion 🤷🏻)

Comment (Maleclypse) [inline data/json/itemgroups/SUS/library.json:207]:
```suggestion
      { "item": "novel_tragedy", "count": [ 1, 5 ], "prob": 80 }
```

File: data/json/items/book/anthology.json [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    "weight": "225 g",
+    "volume": "408 ml",
+    "longest_side": "20 cm",
+    "copy-from": "book_antho_tpl"
+  },
+  {
+    "id": "book_anthology_sunvault",
+    "type": "BOOK",
+    "name": { "str": "Sunvault: Stories of Solarpunk & Eco-Speculation", "str_pl": "copies of Sunvault" },
+    "description": "Sunvault is the first English-language anthology to broadly collect solarpunk short fiction, artwork, and poetry. A new genre for the 21st Century, solarpunk is a revolution against despair. Focusing on solutions to environmental disasters, solarpunk envisions a future of green, sustainable energy used by societies that value inclusiveness, cooperation, and personal freedom.  Published by Upper Rubber Boot Books",
```
Comment (casswedson) [inline data/json/items/book/anthology.json:19]:
```suggestion
    "description": "Sunvault is the first English-language anthology to broadly collect solarpunk short fiction, artwork, and poetry.  A new genre for the 21st Century, solarpunk is a revolution against despair.  Focusing on solutions to environmental disasters, solarpunk envisions a future of green, sustainable energy used by societies that value inclusiveness, cooperation, and personal freedom.  Published by Upper Rubber Boot Books.",
```

File: data/json/items/book/anthology.json [hunk 3]
```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "id": "book_anthology_moataatgb",
+    "type": "BOOK",
+    "name": {
+      "str": "The Museum of All Things Awesome and That Go Boom",
+      "str_pl": "copies of The Museum of All Things Awesome and That Go Boom"
+    },
+    "description": "The Museum of All Things Awesome and That Go Boom is an anthology of science fiction featuring blunt force trauma, explosions, adventure, derring-do, tigers, Martians, zombies, fanged monsters, dinosaurs (alien and domestic), ray guns, rocket ships, and anthropomorphized marshmallows.  Published by Upper Rubber Boot Books",
```
Comment (casswedson) [inline data/json/items/book/anthology.json:9]:
```suggestion
    "description": "The Museum of All Things Awesome and That Go Boom is an anthology of science fiction featuring blunt force trauma, explosions, adventure, derring-do, tigers, Martians, zombies, fanged monsters, dinosaurs (alien and domestic), ray guns, rocket ships, and anthropomorphized marshmallows.  Published by Upper Rubber Boot Books.",
```

File: data/json/items/book/anthology.json [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    "weight": "225 g",
+    "volume": "408 ml",
+    "longest_side": "20 cm",
+    "copy-from": "book_antho_tpl"
+  },
+  {
+    "id": "book_anthology_sst",
+    "type": "BOOK",
+    "name": { "str": "Sharp & Sugar Tooth", "str_pl": "copies of Sharp & Sugar Tooth" },
+    "description": "Sharp & Sugar Tooth: Women Up To No Good is a horror anthology of dark fiction and darker appetites, edited by Octavia Cade. Containing 22 stories of “bad” women, and “good” women who just haven’t been caught yet, it features 22 fearless writers who identify as female, non-binary, or a marginalized sex or gender identity.  Published by Upper Rubber Boot Books",
```
Comment (casswedson) [inline data/json/items/book/anthology.json:29]:
```suggestion
    "description": "Sharp & Sugar Tooth: Women Up To No Good is a horror anthology of dark fiction and darker appetites, edited by Octavia Cade.  Containing 22 stories of “bad” women, and “good” women who just haven’t been caught yet, it features 22 fearless writers who identify as female, non-binary, or a marginalized sex or gender identity.  Published by Upper Rubber Boot Books.",
```

File: data/json/items/book/anthology.json [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    "weight": "225 g",
+    "volume": "408 ml",
+    "longest_side": "20 cm",
+    "copy-from": "book_antho_tpl"
+  },
+  {
+    "id": "book_anthology_bk",
+    "type": "BOOK",
+    "name": { "str": "Broad Knowledge", "str_pl": "copies of Broad Knowledge" },
+    "description": "Broad Knowledge: 35 Women Up To No Good is a feminist anthology of dark fiction and darker knowledge, edited by Joanne Merriam. Containing 35 stories of “bad” women, and “good” women who just haven’t been caught yet.  Published by Upper Rubber Boot Books",
```
Comment (casswedson) [inline data/json/items/book/anthology.json:39]:
```suggestion
    "description": "Broad Knowledge: 35 Women Up To No Good is a feminist anthology of dark fiction and darker knowledge, edited by Joanne Merriam.  Containing 35 stories of “bad” women, and “good” women who just haven’t been caught yet.  Published by Upper Rubber Boot Books.",
```

File: data/json/items/book/anthology.json [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    "weight": "225 g",
+    "volume": "408 ml",
+    "longest_side": "20 cm",
+    "copy-from": "book_antho_tpl"
+  },
+  {
+    "id": "book_anthology_cw",
+    "type": "BOOK",
+    "name": { "str": "Choose Wisely", "str_pl": "copies of Choose Wisely" },
+    "description": "Choose Wisely: 35 Women Up To No Good is a feminist anthology of dark fiction, co-edited by H. L. Nelson and Joanne Merriam. Containing 35 stories of “bad” women, and “good” women who just haven’t been caught yet, it features Joyce Carol Oates, Aimee Bender, Diane Cook, and 33 other fearless women writers.  Published by Upper Rubber Boot Books",
```
Comment (casswedson) [inline data/json/items/book/anthology.json:49]:
```suggestion
    "description": "Choose Wisely: 35 Women Up To No Good is a feminist anthology of dark fiction, co-edited by H. L. Nelson and Joanne Merriam.  Containing 35 stories of “bad” women, and “good” women who just haven’t been caught yet, it features Joyce Carol Oates, Aimee Bender, Diane Cook, and 33 other fearless women writers.  Published by Upper Rubber Boot Books.",
```

File: data/json/items/book/anthology.json [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+    "weight": "225 g",
+    "volume": "408 ml",
+    "longest_side": "20 cm",
+    "copy-from": "book_antho_tpl"
+  },
+  {
+    "id": "book_anthology_an",
+    "type": "BOOK",
+    "name": { "str": "Apocalypse Now", "str_pl": "copies of Apocalypse Now" },
+    "description": "Every society and every generation has its version of the apocalypse: swine flu, genetic mutation, global warming, nuclear fallout, the second coming, peak oil, mass extinction, giant irradiated ants, zombies… Apocalypse Now: Poems and Prose from the End of Days is the first anthology of its kind to bring together the poetry and prose of some of America’s finest (though not always most well-known) literary voices with an eye for the literary and the popular, for story and lyric, for the past and the future, for the psychological and the physical, for the real and the fantastic.  Published by Upper Rubber Boot Books",
```
Comment (casswedson) [inline data/json/items/book/anthology.json:59]:
```suggestion
    "description": "Every society and every generation has its version of the apocalypse: swine flu, genetic mutation, global warming, nuclear fallout, the second coming, peak oil, mass extinction, giant irradiated ants, zombies… Apocalypse Now: Poems and Prose from the End of Days is the first anthology of its kind to bring together the poetry and prose of some of America’s finest (though not always most well-known) literary voices with an eye for the literary and the popular, for story and lyric, for the past and the future, for the psychological and the physical, for the real and the fantastic.  Published by Upper Rubber Boot Books.",
```

File: data/json/items/book/anthology.json [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+    "copy-from": "book_antho_tpl"
+  },
+  {
+    "id": "book_anthology_htloop",
+    "type": "BOOK",
+    "name": {
+      "str": "How to Live on Other Planets: A Handbook for Aspiring Aliens",
+      "str_pl": "copies of How to Live on Other Planets"
+    },
+    "description": "How to Live on Other Planets: A Handbook for Aspiring Aliens explores the immigrant experience in a science fiction setting, with exciting fiction and poetry from some of the genre’s best writers.  Published by Upper Rubber Boot Books",
```
Comment (casswedson) [inline data/json/items/book/anthology.json:72]:
```suggestion
    "description": "How to Live on Other Planets: A Handbook for Aspiring Aliens explores the immigrant experience in a science fiction setting, with exciting fiction and poetry from some of the genre’s best writers.  Published by Upper Rubber Boot Books.",
```

**Pull request merged**